torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2170,8 +2170,8 @@ def sample_inputs_linalg_qr(op_info, device, dtype, requires_grad=False, **kwarg
     # TODO: add 0 to 'ns' and (0, ) to 'batches'
```

Comment:
Does this fix address the cited issue? https://github.com/pytorch/pytorch/issues/50576

Reply:
It does. I forgot to remove this TODO.